docs/release-guide/README.md

```diff
@@ -12,9 +12,28 @@ This is the process we follow when releasing a new version of Cesium for Omniver
 8. Tag the release, e.g., `git tag -a v0.0.0 -m "0.0.0 release"`.
 9. Push the tag to github: `git push origin v0.0.0`.
 10. Wait for the release tag CI build to complete.
-11. Create a new release on GitHub: https://github.com/CesiumGS/cesium-omniverse/releases/new. Copy the changelog into it. Follow the format used in previous releases. Upload the Linux and Windows packages.
+11. Create a new release on GitHub: https://github.com/CesiumGS/cesium-omniverse/releases/new.
+    * Chose the new tag.
+    * Copy the changelog into the description. Follow the format used in previous releases.
+    * Upload the Linux and Windows packages.
+12. Proceed to [Releasing a new version of Cesium for Omniverse Samples](#releasing-a-new-version-of-cesium-for-omniverse-samples).
 
-### Build Linux Package
+# Releasing a new version of Cesium for Omniverse Samples
+
+1. Create a new access token using the CesiumJS ion account.
+    * The name of the token should match "Cesium for Omniverse Samples vX.X.X - Delete on April 1st, 2023" where the version is the same as the Cesium for Omniverse release and the expiry date is two months later than present.
+    * The scope of the token should be "assets:read" for all assets.
```

Comment:
I'm making the assumption here that anything in **CesiumJS** account is intended for public access in demos etc?  

And **CesiumGS** is intended for private data, internal demos, tests, etc?